packages/java/endpoint/src/main/java/dev/hilla/EndpointUtil.java

```diff
@@ -31,7 +31,7 @@
 import org.springframework.web.util.pattern.PathPatternParser;
 
 /**
- * A util class related to {@link Endpoint}.
+ * A util class related to published classes.
```

Comment:
Not sure if `published classes` gives any concrete message about what are we talking about. 
Maybe for now we could say:
```
A utility class related to {@link Endpoint} and {@link BrowserCallable}.
```
or 
```
A utility class related to browser-callable services.
```

Reply:
Reworded